docs/data/release_archive.yml

```diff
@@ -483,6 +483,9 @@ release_archive:
       release_date: 2017-07-27
 
   flink_ml:
+    - version_short: 2.2
+      version_long: 2.2.0
+      release_date: 2023-04-??
```

Comment:
Let's update the release data here.

Reply:
Thanks for the comment. It is fixed now.